dds/DCPS/SafetyProfileStreams.h

```diff
@@ -29,6 +29,7 @@ OpenDDS_Dcps_Export String to_dds_string(long long to_convert);
 OpenDDS_Dcps_Export String to_dds_string(unsigned long long to_convert, bool as_hex = false);
 OpenDDS_Dcps_Export String to_dds_string(unsigned long to_convert, bool as_hex = false);
 OpenDDS_Dcps_Export String to_dds_string(const unsigned char* array, size_t length);
+OpenDDS_Dcps_Export String to_dds_string(const String& to_convert);
```

Comment:
Why is this function needed?  Add a comment if we're keeping it?

Reply:
The thread id can be a string or integer.  In either case, calling `to_dds_string` generically converts it to a string.

Reply:
That makes sense, but I think it should be `inline` and add a comment.